src/CodeCoverage.php

```diff
@@ -909,4 +839,26 @@ private function selectDriver()
             return new PHP_CodeCoverage_Driver_Xdebug;
         }
     }
+
+    /**
+     * Initialise the data with the executable/dead lines from each file in the whitelist
+     */
+    private function initData()
+    {
+        foreach ($this->filter()->getWhitelist() as $file) {
+            $this->driver->start();
+            include_once($file);
```

Comment:
OK. I think I have an idea about how we can resurrect this flag and still improve performance. Leave this with me and I will take a look.

Thanks